src/cookiecutter/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:alpine3.15
+
+ARG COOKIECUTTER_VERSION
+
+RUN apk add --update \
+    bash \
+    py-pip \
+    g++ \
+    git && \
+    pip install cookiecutter==$COOKIECUTTER_VERSION && \
+    apk del g++ py-pip && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir -p /.cookiecutter_replay /.cookiecutters && \
+    chmod 777 /.cookiecutter_replay /.cookiecutters
+
+WORKDIR /app
```

Comment:
This should not be a `VOLUME` instead ? What is the purpose of this `WORKDIR` and how it should be used ?

Reply:
This is where the projects will be generated so you need to run cookiecutter like this:
```bash
docker run --rm -it -u $(id -u):$(id -g) -v $PWD:/app fundocker/cookiecutter gh:organization/repository
```

Reply:
Ok so I think you should also add `VOLUME /app` instruction before this one.

Reply:
I'm not sure I understand what is the point. At build time I have no idea where this will be mounted on the host. So my understanding is that this can only be done at runtime with the `-v` option.

Reply:
This is the purpose of the `VOLUME` instruction. Creating a mount point used by for exemple the `-v` option with the docker CLI https://docs.docker.com/engine/reference/builder/#volume